dimod/discrete/discrete_quadratic_model.py

```diff
@@ -416,6 +419,24 @@ def get_quadratic_case(self, u, u_case, v, v_case):
         return self._cydqm.get_quadratic_case(
             self.variables.index(u), u_case, self.variables.index(v), v_case)
 
+    def add_constraint_as_quadratic(self, terms: Linear, lagrange_multiplier: float,
+                                    constant: float):
+        """
```

Comment:
Should be on the first line, also no more than 80 char wide

Reply:
ok